datamonster_api/lib/datamonster.py

```diff
@@ -256,12 +281,15 @@ def get_data(
         return df
 
     def get_raw_data(self, datasource, **kwargs):
-        """Get data for datasource, providing a raw interface. Use with caution.
+        """Get raw data for all companies available in the data source.
 
-        :param datasource: Datasource object to get the data for
-        :param kwargs: unparsed kwargs to get passed as query parameters
+        :param datasource: ``Datasource`` object to get the data for
+        :param kwargs: unparsed ``kwargs`` to get passed as query parameters
 
         :return: (schema, pandas.DataFrame)
+
+        See `this <examples.html#get-raw-data>`_
```

Comment:
standardize this type of text - above you say "See here for an `example <quickstart.html#>`_."